django_ipam/static/django-ipam/js/hosts-infinite-scrolling.js

```diff
@@ -0,0 +1,87 @@
+/*jslint sloppy: true, browser:true */
```

Comment:
Why was `sloppy: true` required? :smile:

Reply:
It was not after all, forgot to remove it after I solved some issue :smile: